test/lib/package-lock.test.ts

```diff
@@ -116,6 +116,18 @@ test('Parse npm package-lock.json with dev deps only', async (t) => {
   t.deepEqual(depTree, expectedDepTree, 'Tree is created with dev deps only');
 });
 
+test('Parse npm simple package-lock.json dev and prod deps', async (t) => {
+  const expectedDepTree = load('simple-test/expected-tree.json');
+  const depTree = await buildDepTreeFromFiles(
+    `${__dirname}/fixtures/simple-test/`,
+    'package.json',
+    'package-lock.json',
+    true,
+  );
+
+  t.deepEqual(depTree, expectedDepTree, 'Tree is created with dev deps only');
```

Comment:
`dev and prod deps` so should the message here also reflect that it is expected to have both?

Reply:
Updated